library/core/src/num/f128.rs

```diff
@@ -1196,7 +1196,8 @@ impl f128 {
     #[inline]
     #[must_use]
     #[unstable(feature = "f128", issue = "116909")]
-    pub fn total_cmp(&self, other: &Self) -> crate::cmp::Ordering {
+    #[rustc_const_unstable(feature = "const_cmp", issue = "143800")]
```

Comment:
IIRC, you only need `rustc_const_unstable` for things that are otherwise stable but their constness is unstable. You don't need it for things that are already marked `unstable`.

Reply:
That's true. But the question is about ordering. I don't know whether f128 or const_cmp will stabilize first. So I'm being conservative.

Reply:
Using `const_cmp` should be fine here.